tests/parsers/luminex_xponent/testdata/luminex_xPONENT_example01.csv

```diff
@@ -5,7 +5,7 @@ Date,05/17/2023,6:42 PM,,,,,,,,,,,,,,,,,,,,,
 SN,FM3DD12341234,,,,,,,,,,,,,,,,,,,,,,
 Batch,ABCD_1234_20231225,,,,,,,,,,,,,,,,,,,,,,
 Version,1,,,,,,,,,,,,,,,,,,,,,,
-Operator,waldo,,,,,,,,,,,,,,,,,,,,,,
+Operator,nguymaip,,,,,,,,,,,,,,,,,,,,,,

```

Comment:
@LimulusPolyphemus tests are failing because of this change.

Reply:
@slopez-b okay, weird. I don't know that got introduced, but I'll change it back.